server/util/controller_helpers.js

```diff
@@ -23,11 +23,16 @@ export function validCampaignStatusUpdate(prevStatus, currStatus) {
 }
 
 export function allowDraftCampaignUpdates(currentStatus, requestBody) {
+  const { name, title, description, contact_lists_id, script_id } = requestBody;
+  const params = { name, title, description, contact_lists_id, script_id };
   if (currentStatus === 'draft') {
-    const { name, title, description, contact_lists_id, script_id } = requestBody;
     return { name, title, description, contact_lists_id, script_id };
   }
-  throw new Error('Cannot edit campaign that is not in draft.');
+  const isValid = _.every(Object.keys(params), param => !!params[param] === false);
+  if (!isValid) {
+    throw new Error('Cannot edit campaign that is not in draft.');
+  }
+  return true;
```

Comment:
The previous return statement returns an object with update params. This return statement just returns a boolean. We need to be consistent here. How does the calling method use this value?

Reply:
👍